src/components/Input/Input.tsx

```diff
@@ -59,7 +59,6 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>((props, ref?) => {
         value: value,
         "aria-label": ariaLabel,
         "aria-labelledby": ariaLabelledBy,
-        ...attributes,
```

Comment:
Why was this deleted?

Reply:
It was also added directly to the `input` element, so effectively they were added twice.